ipet/parsing/ReaderManager.py

```diff
@@ -273,22 +283,23 @@ def collectData(self):
             readers = [r for r in self.getManageables(True) if r.supportsContext(context)]
 
             line = (0,"")
-            for line in self.testrun:
-                if self.startOfProblemReached(line[1]):
-                    if context in [CONTEXT_ERRFILE, CONTEXT_LOGFILE]:
-                        # .errfiles do not contain problemdexpression ==ready==
+            try:
+                for line in self.testrun:
+                    if self.startOfProblemReached(line[1]):
+                        if context in [CONTEXT_ERRFILE, CONTEXT_LOGFILE]:
+                            # .errfiles do not contain problemdexpression ==ready==
+                            self.finishProblemParsing(line, context, readers)
+                        self.updateProblemName(line, context, readers)
+
+                    if self.endOfProblemReached(line[1]):
                         self.finishProblemParsing(line, context, readers)
-                    self.updateProblemName(line, context, readers)
-
-
-                if self.endOfProblemReached(line[1]):
-                    self.finishProblemParsing(line, context, readers)
-
-                else:
-                    if context == CONTEXT_LOGFILE:
-                        self.activeSolver.readLine(line[1])
-                    for reader in readers:
-                        reader.operateOnLine(line[1])
+                    else:
+                        if context == CONTEXT_LOGFILE:
+                            self.activeSolver.readLine(line[1])
+                        for reader in readers:
+                            reader.operateOnLine(line[1])
+            except IPETInconsistencyError as e:
```

Comment:
Here, you stop for both the out and the error file as soon as something was missing. This is not good. You should simply warn that there are missing parts and skip the remainder **only** in the case of the err file.

Reply:
I changed this